src/api/metadata.ts

```diff
@@ -337,6 +337,43 @@ export class MetadataApi<S extends Schema> {
     );
   }
 
+  /**
+   * Will deploy a recently validated deploy request
+   *
+   * @param options.id = the deploy ID that's been validated already from a previous checkOnly deploy request
+   * @param options.rest = a boolean whether or not to use the REST API
+   */
+  public async deployRecentValidation(options: {
+    id: string;
+    rest?: boolean;
+  }): Promise<string> {
+    const rest = options.rest;
+    delete options.rest;
```

Comment:
Doesn't look like you need to delete `options.rest`.  You can probably just do:
`const { id, rest } = options;`
and use them below.